rvip/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rvip)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  tf
+  rospy
+  roscpp
+  roslint
+  pcl_ros
+  std_msgs
+  cv_bridge
+  sensor_msgs
+  geometry_msgs
+  image_transport
+  visualization_msgs
+)
+
+roslint_cpp()
+
+# DEPENDENCIES
+find_package(OpenGR REQUIRED PATHS /usr/local/lib/cmake)
```

Comment:
Afaik, `/usr/local/lib/cmake` is on the default search path for CMake. Is this really needed?

What if users have OpenGR installed somewhere else?

Reply:
If users follow the **OpenGR** setup instructions, **OpenGR** should be installed under **CMake**.

The line would be needed to ensure it is indeed installed, following the instructions given.

Reply:
`/usr/local` is already on the search path of CMake (see [CMAKE_SYSTEM_PREFIX_PATH](https://cmake.org/cmake/help/v3.0/variable/CMAKE_SYSTEM_PREFIX_PATH.html)).

This makes me wonder whether it's really necessary to explicitly add it to `PATHS` here.